sdks/go/internal/json_utils.go

```diff
@@ -0,0 +1,55 @@
+package internal
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+)
+
+func ReadValueFromRequest(r *http.Request) (any, error) {
+	//Decode body into catena value type
+	var v any
+
+	if err := json.NewDecoder(r.Body).Decode(&v); err != nil {
+		return nil, err
+	}
+	//Convert catena value type to go native type
+	switch val := v.(type) {
+	case map[string]interface{}:
```

Comment:
interface{} can be replaced by any (from the go extension)
```suggestion
	case map[string]any
```